Automattic/wp-calypso
# Guided Tours: Remove obsolete jetpackBackupsRewind tour

#### Changes proposed in this Pull Request

While reviewing #52365 I noticed that the `jetpackBackupsRewind` can no longer be triggered, because the triggering elements no longer exist. Seems like this has been the case for almost 2 years - the old credentials form was removed in #37488. Since then, that tour can is no longer relevant and can't be triggered at all. So it can be completely removed.

#### Testing instructions

* Verify that the element that triggers the first tour step is missing.
* Verify that the tour isn't used anywhere.

#### Context

Noticed while reviewing #52365.

Closes #52365.

Related to #37488
